modules/llm-cache/radix-tree/radix-tree.cc

```diff
@@ -210,10 +209,6 @@ std::string RadixTree::Serialize() {
   // raxShow(this->tree);
   std::string serializedStr;
 
-  if (tokenList.size() != dataList.size()) {
-    throw std::runtime_error(
-        "The size of token list and data list is not equal");
-  }
```

Comment:
Just sliently ignore the error??

Reply:
It is not an error.
In serialize, If it push one element to tokenlist, it must also push a element to datalist.